### DISCUSSION TITLE
Create graphs and charts on home page

### DISCUSSION THREAD

**Discussion by unfazedEgnimadevspace** [unfazedEgnimadevspace/Gracker#51] [Ideas]
Hello @americodls 

How about we help users with charts and graphs on the app_root page,  so when they log in successfully they are greeted with a visual representation of their progress  on their various exercises and calories recorded.

What do you think about this idea?

Thank you

**Comment by unfazedEgnimadevspace** [comment]
![dashboard](https://github.com/unfazedEgnimadevspace/Gracker/assets/96634477/5e17d09e-93b0-4d40-ba9d-4b417ab68f41)


Hello @americodls 

I already this have this implemented, should i make a pull request so you  can check and we can possibly  merge it or there is anything you will like to change from the screenshot

**Reply by americodls** [reply]
If you want feedback on your code a pull request will be the best option.
But if you don't you can go ahead and merge it. No problem.
